recipes/wxpython/build-linux.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
```

Comment:
I'd rather leave it in -- again, if anyone want to pick this up, it will be there as documentation.